services/ui/src/routes/index.svelte

```diff
@@ -1 +1,15 @@
-<p>This space intentionally blank.</p>
+<script>
+  import VariableDiag from "$lib/components/VariableDiag";
+
+  let data = fetch("http://localhost:5000/diag/temperature").then((response) =>
+    response.json()
+  );
+</script>
+
+{#await data}
+  <p>Loading</p>
+{:then diag}
+  <VariableDiag title="Temperature" {...diag} />
+{:catch}
+  <p>Oops</p>
```

Comment:
I ran into this when I specified the wrong data directory for the Flask app and appreciated it. :sweat_smile: